### PR TITLE
still remove the federated share even if we cant notify the remote

### PR DESCRIPTION
**Pull request by icewind1991**
For #3766

@esprit-libre @Gavisann @vojeroen can you verify if this fixes the problem for you

### CODE REVIEW

**Review by schiessle — APPROVED** [review]
looks good. Ideally we would re-try to send the feedback in a background job later again, but for now I think this fix is good

**Comment by schiessle** [comment]
Probably something we should backport.

**Comment by vojeroen** [comment]
@icewind1991 unfortunately I don't have the possibility to apply a non-release version on our systems

File: apps/files_sharing/lib/External/Manager.php [hunk 1]
```diff
@@ -366,8 +366,13 @@ public function removeShare($mountPoint) {
 		$result = $getShare->execute(array($hash, $this->uid));
 
 		if ($result) {
-			$share = $getShare->fetch();
-			$this->sendFeedbackToRemote($share['remote'], $share['share_token'], $share['remote_id'], 'decline');
+			try {
+				$share = $getShare->fetch();
+				$this->sendFeedbackToRemote($share['remote'], $share['share_token'], $share['remote_id'], 'decline');
+			} catch (\Exception $e) {
```
Comment (rullzer) [inline apps/files_sharing/lib/External/Manager.php]: \Exception might be a bit generic... but i guess for now OK.

I do agree with @schiessle that we should probabaly inject some background job that will try to notify the remote a few times.

**Review by rullzer — APPROVED** [review]

**Pull request merged**